docker-compose.yml.dist

```diff
@@ -9,6 +9,9 @@ services:
     image: sciapp/sampledb:0.20.0
     restart: always
     container_name: sampledb
+    depends_on:
+      db:
+        condition: service_healthy
```

Comment:
The ["long syntax"](https://docs.docker.com/compose/compose-file/#depends_on) of `depends_on` is only available in docker-compose versions >= 1.27.0. For example debian stable only packages 1.25.0 and thus fails to start the container with these changes. Maybe it would make sense to stick with the legacy [`depends_on`](https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on) for now?

Reply:
I see two approaches:

1. use the shorthand, and hope that the implementation will wait for service to be healthy (MAY vs MUST)
2. add a comment on top of that line to indicate the shorthand syntax for users of old versions of docker-compose. Users of old versions would then have the possibility to upgrade their docker-compose (it's standalone so can be curl'd into the system easily) or use the shorthand syntax and introduce the possibility of a failing restart as it might not wait for db to be healthy, but just started.

In my opinion, 2 is better because I really don't see the appeal of using a quite old version of docker-compose, unless one enjoys lacking features and random bugs like this popping up. But that's my approach, I can totally understand if you'd prefer 1. Let me know what you prefer and I'll push a change.

Reply:
I think the second approach is fine. For me the "appeal" of using an older version of docker-compose was that I simply typed `apt install docker-compose` and it worked, so I never bothered to update to a newer version. But I think a comment on the incompatibilities with older docker-compose versions and a short note on how to resolve/work-around them would be enough to help those users to get started :)

Reply:
New commit pushed!